_posts/2015-10-29-code-talks-java.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "code.talks: code & nachos"
+date: "2015-10-29 09:00:00"
+icon: comments
+categories: conferences
+authors: ["Erik"]
+---
+
+Some weeks ago we attended the `code.talks` developer conference in Hamburg.
+Besides the amazing location (which was the Cinemaxx Dammtor) and the awesome catering (nachos, popcorn and Mate as much as you can endure) the conference was also (who would have guessed that) packed with lots of interesting talks and people.
+
+In the variety of talks we found things we're already doing right (according to the speakers) and other aspects we can improve on.
```

Comment:
Listening to the many great presentations we found things that we're already doing right (according to the speakers) but also other aspects we can improve on.